python/pytests/conftest.py

```diff
@@ -30,7 +30,7 @@ def jsonl(self, data: Union[kd.Timestream, pd.DataFrame]) -> None:
         filename = self._filename("jsonl")
 
         if self._save:
-            df.to_json(filename, orient="records", lines=True, date_format="iso")
+            df.to_json(filename, orient="records", lines=True, date_format="iso", date_unit="ns")
```

Comment:
Tweak tests to use `ns` as the unit -- this changes the precision in a lot of golden files, hence the churn, but it was necessary for the golden test comparison to pass in the new tests.